sympy/core/basic.py

```diff
@@ -840,7 +840,10 @@ def subs(self, *args, **kwargs):
         sequence = list(sequence)
         for i in range(len(sequence)):
             o, n = sequence[i]
-            so, sn = sympify(o), sympify(n)
+            try:
+                so, sn = sympify(o), sympify(n)
+            except:
```

Comment:
:x: We (almost) never include a "bare except" since the code in the try-clause could fail for a number of reasons. Include the expected error in the except-line as `except ValueError` or (for multiple possibilities) `except (ValueError, TypeError)`.